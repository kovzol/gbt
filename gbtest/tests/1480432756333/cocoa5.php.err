0.00user 0.00system 0:00.00elapsed 88%CPU (0avgtext+0avgdata 10092maxresident)k
0inputs+8outputs (0major+675minor)pagefaults 0swaps
