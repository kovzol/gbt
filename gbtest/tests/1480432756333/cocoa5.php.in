use QQ[v5,v6], DegRevLex;
GBasis(ideal(2+5*v6+-1*v6^2+2*v5+-1*v5^2));
