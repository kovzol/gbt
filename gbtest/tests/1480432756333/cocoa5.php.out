*** Cannot access default package directory: "packages" ***

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!! WARNING: Standard CoCoA packages not found !!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

   ______      ______      ___         ______
  / ____/___  / ____/___  /   |       / ____/
 / /   / __ `/ /   / __ `/ /| |______/___ `  
/ /___/ /_/ / /___/ /_/ / ___ /_____/___/ /  
`____/`____/`____/`____/_/  |_|    /_____/   
With CoCoALib and external libraries CDD FROBBY GFAN GSL NORMALIZ READLINE 
indent(VersionInfo()); -- for information about this version

# use QQ[v5,v6], DegRevLex;

# GBasis(ideal(2+5*v6+-1*v6^2+2*v5+-1*v5^2));
[-v5^2 -v6^2 +2*v5 +5*v6 +2]

# 